analyzer/tools/compilation-database-transformer/compilation_database_transformer/cli.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+Manipulate, preprocess and transform compilation database JSON files.
+Provides pretty printing, compilation result checking and adjusting
+compilation parameters features.ccdb-tool is used to manipulate
+compile command databases.
+"""
+
+import argparse
+import operator
+import shlex
+import subprocess
+import sys
+
+from compilation_database_transformer.build_action import BuildAction
```

Comment:
If I run this tool from the CodeChecker build directory I will get the following error message:
```
Traceback (most recent call last):
  File "~/codechecker/build/CodeChecker/bin/ccdb-tool", line 20, in <module>
    from compilation_database_transformer.build_action import BuildAction
ModuleNotFoundError: No module named 'compilation_database_transformer'
```

Please check the report-converter tool how we solved this:
https://github.com/Ericsson/codechecker/blob/74540cb744606992eefa37363b695573994a8ae4/tools/report-converter/codechecker_report_converter/cli.py#L14-L25

Reply:
Thanks, I will fix this accordingly!